backstop.json

```diff
@@ -158,6 +158,23 @@
       "selectorExpansion": true,
       "misMatchThreshold": 1.0,
       "requireSameDimensions": true
+    },
+    {
+      "label": "mention_box_highlighter_for_mentions",
+      "cookiePath": "",
+      "url": "http://localhost:9001/iframe.html?selectedKind=Mention%20box&selectedStory=Highlighter%20for%20mentions",
```

Comment:
Are you sure about this hardcoded link to localhost? I don't know how Backstop works (it might be perfectly fine to have this here) but it seems a bit off, so I'm drawing some focus to it.

Reply:
As long as the test will be ran on the local machine I guess it should be localhost - not trying to host it anywhere. When running tests the setup launches both the mock server and backstop on localhost.

Reply:
Sure, then it's fine.